Gemfile

```diff
@@ -16,6 +17,7 @@ gem 'jquery-rails'
 gem 'jquery-turbolinks'
 gem 'newrelic_rpm'
 gem 'omniauth-myusa', git: 'https://github.com/18F/omniauth-myusa.git'
+gem "paperclip", "~> 4.2"
```

Comment:
There are differing opinions on this, but I would leave the version restriction off unless there's a specific reason to lock the range.